.github/scripts/upload_tests_results.py

```diff
@@ -0,0 +1,342 @@
+                    "status": status,
+                    "status_description": status_description.replace("\r\n", ";;").replace("\n", ";;").replace("\"", "'"),
+                    "log": "" if testcase.find("properties/property/[@name='url:log']") is None else testcase.find("properties/property/[@name='url:log']").get('value'),
+                    "logsdir": "" if testcase.find("properties/property/[@name='url:logsdir']") is None else testcase.find("properties/property/[@name='url:logsdir']").get('value'),
+                    "stderr": "" if testcase.find("properties/property/[@name='url:stderr']") is None else testcase.find("properties/property/[@name='url:stderr']").get('value'),
+                    "stdout": "" if testcase.find("properties/property/[@name='url:stdout']") is None else testcase.find("properties/property/[@name='url:stdout']").get('value'),
+
+                }
+            )
+    return results
+
+
+def get_codeowners_for_tests(entries, tests_data):
+    tests_data_with_owners = []
+    for test in tests_data:
+        owners = []
+        target_path = test["suite_name"]
+        for path, path_owners in entries:
+            path = path + "/" if path[-1] != "/" else path
+            path = path + "*" if path[-1] != "*" else path
+            if fnmatch.fnmatch(f"/{target_path}", path):
+                owners.extend(path_owners)
+
+        test["owners"] = ",".join(owners)
+        tests_data_with_owners.append(test)
+    return tests_data_with_owners
+
+
+def upload_codeowners(session, entries):
+    # Вставка данных в таблицу
+    for entry in entries:
+        path, owners = entry
+        sql = f"""
+        UPSERT INTO codeowners (path, owners) VALUES 
+            ("{path}", {str(owners).replace("'", '"')});
+        """
+        session.transaction().execute(sql, commit_tx=True)
+
+
+def create_table(session, sql):
+
+    try:
+        session.execute_scheme(sql)
+    except ydb.issues.AlreadyExists:
+        pass
+    except Exception as e:
+        print(f"Error creating table: {e}, sql:\n{sql}", file=sys.stderr)
+        raise e
+
+
+# Создание таблицы
+def create_codeowners_table(session, table_path):
+    sql = f"""
+    --!syntax_v1
+    CREATE TABLE IF NOT EXISTS `{table_path}` (
+        path Utf8,
+        owners Utf8,
+        PRIMARY KEY (path)
+    );
+    """
+    create_table(session, sql)
+
+
+def create_tests_table(session, table_path):
+    # Создание таблицы, если ее еще нет
+    sql = f"""
+    --!syntax_v1
+    CREATE TABLE IF NOT EXISTS `{table_path}` (
+        build_type Utf8,
+        job_name Utf8,
+        job_id Uint64,
+        commit Utf8,
+        branch Utf8,
+        pull Utf8,
+        run_time Timestamp,
+        test_id Utf8,
+        suite_name Utf8,
+        raw_test_name Utf8,
+        file Utf8,
+        test_name Utf8,
+        fixture Utf8,
+        time Double,
+        status Utf8,
+        status_description Utf8,
+        owners Utf8,
+        log Utf8,
```

Comment:
Logs could be huge, i am not sure that it is a good idea to put logs as value in ydb

Reply:
Also they are empty in db: https://console.yandex.cloud/folders/b1g0a77t7u7b81f3pu4o/ydb/databases/etnvsjbk7kh1jc6bbfi8/browse?path=%2Ftest_results%2Fpr_check&page=2&pageSize=100

Reply:
this var for URL for logs in s3